tests/phpunit/tests/theme.php

```diff
@@ -9,7 +9,7 @@ class Tests_Theme extends WP_UnitTestCase {
 	protected $theme_slug = 'twentysixteen';
 	protected $theme_name = 'Twenty Sixteen';
 	protected $default_themes = array(
-		'twentyfifteen', 'twentysixteen', 'twentyseventeen',
+		'twentyfifteen', 'twentysixteen', 'twentyseventeen', 'cp2017-child', 'cp2016-child'
```

Comment:
Let's not include this change in this PR, it's part of #292 instead.

Reply:
Sorry for that, I don't even know how this got in here?

Now how to get it out?